### ISSUE TITLE
Principle #<ENTER NUMBER HERE> "<ENTER NAME HERE>"

### ISSUE THREAD

**Issue by nataled** [OBOFoundry/OBOFoundry.github.io#1041]
See #1040

We need be concerned when evaluating license that there might hidden issues with ontologies imported by the ontology of interest.

@cmungall

**Comment by nataled** [comment]
Proposal: We should look at the prefixes found in various ontologies. If there are any imports from non-OBO Foundry ontologies, we need to be mindful that there might be license issues. Note that this won't handle copied (as opposed to MIREOTed or OWL imported) terms.

Possible automated check: look for URIs that are in axioms, then check the license for any ontologies mentioned.

**Comment by nlharris** [comment]
what's the status of this? (also flagging @kltm in case he has comments)

**Comment by nlharris** [comment]
I guess this can't be closed until the unmerged PRs mentioned above are merged?

**Comment by nataled** [comment]
I don't think this depends on those unmerged PRs. Rather, it awaits some tooling that will help determine if there are possible issues.

**Comment by matentzn** [comment]
I think this is both an important and a scary issue. I have been getting some help from @kltm understanding this for another problem (mapping files) here https://github.com/mapping-commons/sssom/issues/132

We do _not_ want to get into the business of interpreting license agreements. I would argue against automated checks, because this may cause the totally false expectation that we may know what we are doing, and if it is green, OBO Foundry says its all good. Its not. We should never do that. But what we should do is to think about how to improve our automated pipelines in ODK etc with _attribution_ features. For example, we could preserve the licenses of imported ontologies in the ontology header in some way, and make sure we don't lose attribution. Just something to think about. @nlharris this ticket likely wont close until 2030, so no need to worry about it :P

**Comment by nlharris** [comment]
I'll make myself a calendar reminder to circle back in 2030. 😉
